sdk/resourcemanager/Azure.ResourceManager.Core/docs/MigratingFromCurrentToPreview.md

```diff
@@ -0,0 +1,86 @@
+# Differences between current and preview .NET Management SDK
+
+## Structured Resource Identifier
```

Comment:
I don't feel like the breakdown of resourceid types is something we want to highlight in a migration guide.  The main things we should highlight are:
- SubscriptionIds contain useful properties like 'subscription' and 'resourceGroup', appropriate to the resource they identify (i.e. resources homed in a tenant will not have a subscriptionId property)
- resourceId and string are essentially interchangeable - I can pass in a string in place of a resource Id and the framework will automatically parse and convert the resource id string.
- Any resourceId returned from one of our methods will *automatically* have

Reply:
Agreed, I am going to dismiss the comments about the migration guides since those will be added in a future PR.